### DISCUSSION TITLE
help to create kick commands

### DISCUSSION THREAD

**Discussion by jkrilin** [Sehelitar/Kick.bot#84] [Q&A]
hello i just discovered this function and i am trying to make the commands i put to work on kick but i can't get it to work or i don't know how to do it. i am looking everywhere but it doesn't appear but i put it to play on twitch and it works. i use the latest version of streamer.bot. can you help me or give me an idea? thanks.

**Comment by mocchimon** [comment]
First you need to create an command then the action. The trigger will be the command

![image](https://github.com/user-attachments/assets/cfed77b8-0982-4323-bf1e-2574c080acd4)

**Reply by mocchimon** [reply]
![image](https://github.com/user-attachments/assets/4f4f06cc-2efd-4f81-8da7-a09edf1c3759)
Here's a kick example
